app/google-services.json

```diff
@@ -0,0 +1,48 @@
+{
```

Comment:
This file should also be added to the .gitignore; we can distribute it locally at the next meeting.